### PR TITLE
feat(processes): document time goals feature

### PR DESCRIPTION
**Pull request by OmranAbazid**
related to OPT-6088

### CODE REVIEW

**Comment by akeller** [comment]
@OmranAbazid is this ready for review? Is there a technical reviewer on the Optimize side for this topic? 

Please add @christinaausley as a reviewer when you are ready for a technical style guide review.

File: docs/components/optimize/userguide/processes.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+id: processes
+title: Processes
+---
+
+
+## Overview
+
+The processes section give you an overview of all the processes and their status on a single page.
```
Comment (megglos) [inline docs/components/optimize/userguide/processes.md:9]:
```suggestion
The processes section gives you an overview of all the processes and their status on a single page.
```

File: docs/components/optimize/userguide/processes.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+It allows you to set a process owner to take responsibility of the process and time goals to track its performance.
+
+![Processes page](./img/processes.png)
+
+
+
+## Setting an owner: 
+Assign the responsibility for process health to have the right conversation with the right person to fix a process. Once you open up the set owner modal, you can select one of the users in your organization to be the main person responsible for the process.
```
Comment (megglos) [inline docs/components/optimize/userguide/processes.md:17]: nit: just a matter of taste, I don't think we need to highlight a reason but rather just indicate the meaning of an owner? wdyt about this?
```suggestion
Assign the owner for process health to make transparent who is the responsible individual for it. Once you open up the set owner modal, you can select one of the users in your organization to be the main person responsible for the process.
```

File: docs/components/optimize/userguide/processes.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+![Set Owner](./img/set-owner.png)
+
+Once an owner is set on a process, You can sort the processes page by owner to see how each of your process owners perform.
```
Comment (megglos) [inline docs/components/optimize/userguide/processes.md:21]: also just a matter of taste, I think it's not really about owner performance but neat to get a quick overview of processes by owner?
```suggestion
Once an owner is set on a process, You can sort the processes page by owner to quickly see which processes have a common owner.
```

File: docs/components/optimize/userguide/processes.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+## Set and track time-based goals:
+Setting duration goals on a process makes it easier to quickly see its health over time. It is also useful to track the result of any improvements done on the process.
+
+Once you open up the modal, you’ll be met by two goals: a target goal and an SLA goal for process instance duration. The target goal assumes that 75% of your process instances follow your happy path or can automatically be resolved entirely. In contrast, the SLA goal is to ensure that you can still guarantee your customers how long a process should take while excluding outliers, even if human intervention is required.
+
+![Set time goals](./img/timeGoals.png)
+
```
Comment (megglos) [inline docs/components/optimize/userguide/processes.md:29]:
```suggestion
```

File: docs/components/optimize/userguide/processes.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+## Overview
+
+The processes section give you an overview of all the processes and their status on a single page.
+It allows you to set a process owner to take responsibility of the process and time goals to track its performance.
+
+![Processes page](./img/processes.png)
+
+
```
Comment (megglos) [inline docs/components/optimize/userguide/processes.md:14]:
```suggestion
```

**Review by megglos — COMMENTED** [review]
LGTM, just a couple of suggestions, let me know what you think, we can assign it to Christina afterwards.

**Review by christinaausley — APPROVED** [review]
LGTM on my end. Technical adjustments made in latest commit, including:
- Meta description added (this also helps when sharing a link, so the preview won't default to the first thing on the page (a header).
- Removing `:` from headers -- typically text only is best.
- Clarifying "happy path" on first reference in the case the user is unfamiliar with term.

**Comment by OmranAbazid** [comment]
> LGTM on my end. Technical adjustments made in latest commit, including:
> 
> * Meta description added (this also helps when sharing a link, so the preview won't default to the first thing on the page (a header).
> * Removing `:` from headers -- typically text only is best.
> * Clarifying "happy path" on first reference in the case the user is unfamiliar with term.

Awesome!, Thanks @christinaausley

**Pull request merged**